SWADroid/src/main/res/layout/button_quadrant.xml

```diff
@@ -0,0 +1,208 @@
+                android:orientation="horizontal"
+                android:visibility="gone">
+
+                <ToggleButton
+                    android:id="@+id/button5"
+                    android:layout_width="0dp"
+                    android:layout_height="match_parent"
+                    android:layout_margin="5dp"
+                    android:layout_weight="1"
+                    android:background="@drawable/togglebuttone"
+                    android:text='e'
+                    android:textAllCaps="false"
+                    android:textColor="#ffffff"
+                    android:textOff='e'
+                    android:textOn='e'
+                    android:textSize="30dp" />
+
+                <ToggleButton
+                    android:id="@+id/button6"
+                    android:layout_width="0dp"
+                    android:layout_height="match_parent"
+                    android:layout_margin="5dp"
+                    android:layout_weight="1"
+                    android:background="@drawable/togglebuttonf"
+                    android:text='f'
+                    android:textAllCaps="false"
+                    android:textColor="#ffffff"
+                    android:textOff='f'
+                    android:textOn='f'
+                    android:textSize="30dp" />
+            </LinearLayout>
+
+            <LinearLayout
+                android:id="@+id/linearLayout78"
+                android:layout_width="match_parent"
+                android:layout_height="0dp"
+                android:layout_weight="1"
+                android:orientation="horizontal"
+                android:visibility="gone">
+
+                <ToggleButton
+                    android:id="@+id/button7"
+                    android:layout_width="0dp"
+                    android:layout_height="match_parent"
+                    android:layout_margin="5dp"
+                    android:layout_weight="1"
+                    android:background="@drawable/togglebuttong"
+                    android:text='g'
+                    android:textAllCaps="false"
+                    android:textColor="#ffffff"
+                    android:textOff='g'
+                    android:textOn='g'
+                    android:textSize="30dp" />
+
+                <ToggleButton
+                    android:id="@+id/button8"
+                    android:layout_width="0dp"
+                    android:layout_height="match_parent"
+                    android:layout_margin="5dp"
+                    android:layout_weight="1"
+                    android:background="@drawable/togglebuttonh"
+                    android:text='h'
+                    android:textAllCaps="false"
+                    android:textColor="#ffffff"
+                    android:textOff='h'
+                    android:textOn='h'
+                    android:textSize="30dp" />
+            </LinearLayout>
+
+            <LinearLayout
+                android:id="@+id/linearLayout910"

```

Comment:
You should use a more descriptive name for the id. This improves the readability of the code that refers to this id and makes it easier to identify which button is being used.